src/Command/RoutesCommand.php

```diff
@@ -57,7 +88,16 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
      */
     public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionParser
     {
-        $parser->setDescription('Get the list of routes connected in this application.');
+        $parser
+            ->setDescription('Get the list of routes connected in this application.')
+            ->addOption('verbose', [
+                'help' => 'Display verbose output',
+                'short' => 'v',
+            ])
+            ->addOption('sort', [
+                'help' => 'Sorts alphabetically by route name A-Z',
+                'short' => 's',
```

Comment:
```suggestion
                'short' => 's',
                'boolean' => true,
```

Then `$args->getOption('sort')` will be `true` or `false`

Reply:
What's the downside of a simple null check?

Reply:
Booleans doesn't have value.

Option now consumes next argument as its value

`bin/cake routes --sort --verbose` will set `sort=--verbose`...